package.json

```diff
@@ -20,12 +20,19 @@
   },
   "homepage": "https://github.com/SE2Project-BHKPTZ/backend#readme",
   "dependencies": {
+    "18": "^0.0.0",
```

Comment:
What are these dependencies?

Reply:
I dont know where the came from but they are not needed